docs/expo_development.md

```diff
@@ -27,41 +27,38 @@ Your application crashes in Expo Go when some native code (that is not included)
 
 ```typescript
 // mockDatadog.ts
-import { NativeModules } from 'react-native';
+import {
+    DdLogs,
+    DdTrace,
+    DdRum,
+    DdSdkReactNative
+} from '@datadog/mobile-react-native';
 
 if (__DEV__) {
     const emptyAsyncFunction = () => new Promise<void>(resolve => resolve());
 
-    NativeModules.DdSdk = {
-        initialize: emptyAsyncFunction,
-        setUser: emptyAsyncFunction,
-        setAttributes: emptyAsyncFunction,
-        setTrackingConsent: emptyAsyncFunction
-    };
-
-    NativeModules.DdLogs = {
-        debug: emptyAsyncFunction,
-        info: emptyAsyncFunction,
-        warn: emptyAsyncFunction,
-        error: emptyAsyncFunction
-    };
-
-    NativeModules.DdTrace = {
-        startSpan: emptyAsyncFunction,
-        finishSpan: emptyAsyncFunction
-    };
-
-    NativeModules.DdRum = {
-        startView: emptyAsyncFunction,
-        stopView: emptyAsyncFunction,
-        startAction: emptyAsyncFunction,
-        stopAction: emptyAsyncFunction,
-        addAction: emptyAsyncFunction,
-        startResource: emptyAsyncFunction,
-        stopResource: emptyAsyncFunction,
-        addError: emptyAsyncFunction,
-        addTiming: emptyAsyncFunction
-    };
+    DdLogs.debug = emptyAsyncFunction;
```

Comment:
how do we maintain it if we add new function to the public API of the SDK? Should we have a checklist before the release to update this part of the doc?

Reply:
As discussed, we should push for users to use Expo Development builds. I've added a notice to the file that it might not be up to date.